packages/loaders/src/Progress.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+
+const COMPONENT_ID = 'loaders.progress';
+
+const { zdColorBlue600, zdColorGrey200, zdSpacingXs } = require('@zendeskgarden/css-variables');
+
+const heights = {
+  small: 2,
+  large: 12,
+  default: 6
+};
+
+const sizeToHeight = size => heights[size || 'default'];
+const sizeToBorderRadius = size => sizeToHeight(size) / 2;
+
+const ProgressBarContainer = styled.div.attrs(props => ({
+  borderRadius: sizeToBorderRadius(props.size)
+}))`
+  margin: ${zdSpacingXs} 0;
+  border-radius: ${props => props.borderRadius}px;
+  background: ${zdColorGrey200};
+`;
+
+const ProgressBarIndicator = styled.div.attrs(props => ({
+  height: sizeToHeight(props.size),
+  borderRadius: sizeToBorderRadius(props.size)
+}))`
+  border-radius: ${props => props.borderRadius}px;
+  background: ${zdColorBlue600};
```

Comment:
In order to more easily handle color changes, you might set `color: ${zdColorBlue600}` on the `ProgressBarContainer` and then set `background-color: currentColor` here on the `ProgressBarIndicator`.